MovieApp/Screens/More/Views/SharedCell/SharedTableViewCell.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  SharedTableViewCell.swift
+//  MovieApp
+//
+//  Created by MinhDev on 10/20/21.
+//
+
+import UIKit
+
+protocol sharedDelegate: AnyObject {
+	func sharedFacebook()
+	func sharedGmail()
+	func sharedWhatApps()
+	func sharedAlert()
+}
+
+class SharedTableViewCell: UITableViewCell {
+
+// MARK: - Iboutlets
+	@IBOutlet private weak var labelShared1: UILabel!
+	@IBOutlet private weak var labelShared2: UILabel!
+	@IBOutlet private weak var labelShared3: UILabel!
```

Comment:
```suggestion
	@IBOutlet private weak var shareFacebookLabel: UILabel!
```
Don't using 1,2,3... in variables

Reply:
fix complete